test/resources/live-photos.test.ts

```diff
@@ -0,0 +1,90 @@
+import { Applicant, LivePhoto } from "onfido-node";
+
+import { createReadStream, readFileSync } from "fs";
+
+import {
+  onfido,
+  getExpectedObject,
+  createApplicant,
+  cleanUpApplicants
+} from "../test-helpers";
+
+const exampleLivePhoto: LivePhoto = {
+  id: "123-abc",
+  created_at: "2020-01-01T00:00:00Z",
+  href: "/v3.6/live_photos/123-abc",
+  download_href: "/v3.6/live_photos/123-abc/download",
+  file_name: "sample_photo.png",
+  file_type: "image/png",
+  file_size: 395_856
+};
+
+function getExpectedLivePhoto(exampleLivePhoto: LivePhoto) {
+  return getExpectedObject(exampleLivePhoto, {
+    download_href: expect.stringMatching(
+      /^\/v3.6\/live_photos\/[0-9a-z-]+\/download$/
+    )
+  });
+}
+
+let applicant: Applicant;
+
+beforeEach(async () => {
+  applicant = (await createApplicant()).data;
+});
+
+afterAll(() => {
+  return cleanUpApplicants();
+});
+
+async function uploadLivePhoto(
+  applicant: Applicant,
+  advancedValidation?: boolean
+) {
+  let readStream: any = createReadStream("test/media/sample_photo.png");
+
+  return onfido.uploadLivePhoto(applicant.id, readStream, advancedValidation);
+}
+
+it("uploads a live photo", async () => {
+  const photo = await uploadLivePhoto(applicant);
+
+  expect(photo.data).toEqual(getExpectedLivePhoto(exampleLivePhoto));
+});
+
+// [SKIP] Need to cast advancedValidation parameter to String in default-api.ts:2578:
+// localVarFormParams.append('advanced_validation', String(advancedValidation as any));
+it.skip("uploads a live photo without advanced validation", async () => {
```

Comment:
This test is skipped waiting for a fix on OpenAPI generator (PR will come soon).

Reply:
[PR#18084](https://github.com/OpenAPITools/openapi-generator/pull/18084) and [issue#18083](https://github.com/OpenAPITools/openapi-generator/issues/18083) have been created to address this problem.